Server/Server.Core/Server.Core.Static/SettingsManager.hpp

```diff
@@ -1,34 +1,54 @@
 #pragma once
 
 #include "Settings.hpp"
+#include "ArgParser.hpp"
 
 namespace Server::Builder
 {
+/**
+ * @class SettingsManager
+ * @brief Class for managing server builder settings.
```

Comment:
Class provider of ServerBuilder settings. Contain high level logic that transform base parameters

Reply:
Done.